Julia-co-calling-graph/Julia-co-calling-graph.jl

```diff
@@ -2,10 +2,10 @@ using Pkg.Pkg.TOML
 using Printf
 
 # directory where Julia packages are stored
-pkg_dir = "/Users/mengliu/.julia/packages/"
+pkg_dir = "/Users/vaastavarora/.julia/packages/"
 
 # this variable needs to be changed based on your own installation directory
-path = "/Users/mengliu/.julia/registries/General"
+path = "/Users/vaastavarora/.juliapro/JuliaPro_v1.2.0-2/registries/JuliaPro"
```

Comment:
There should be some way of getting this directory out of the Julia Pkg system...